src/issues/components/IssueTabsUI.vue

```diff
@@ -0,0 +1,153 @@
+            </div>
+          </div>
+          <div class="q-ma-sm q-pa-sm bg-white">
+            <span class="text-bold text-secondary uppercase">{{ issue.createdBy.displayName }}</span>
+            <span class="message-date">
+              <small class="text-weight-light">
+                <DateAsWords :date="issue.createdAt" />
+              </small>
+            </span>
+            <Markdown :source="issue.topic" />
+          </div>
+        </QCollapsible>
+        <ChatConversation
+          v-if="conversation"
+          :conversation="conversation"
+          :away="away"
+          :current-user="currentUser"
+          :start-at-bottom="true"
+          :inline="inline"
+          @send="$emit('send', arguments[0])"
+          @mark="$emit('mark', arguments[0])"
+          @toggleReaction="$emit('toggleReaction', arguments[0])"
+          @saveMessage="$emit('saveMessage', arguments[0])"
+          @fetchPast="$emit('fetchPast', arguments[0])"
+          @fetchFuture="$emit('fetchFuture')"
+        />
+      </QTabPane>
+      <QTabPane name="vote">
+        <IssueVote
+          v-if="issue.status === 'ongoing'"
+          :issue="issue"
+          @saveScores="$emit('saveScores', arguments[0])"
+        />
+        <IssueResults
+          v-if="issue.status !== 'ongoing'"
+          :issue="issue"
+        />
+        <QList
+          v-if="multipleVotings"
+        >
+          <IssueHistoryItem
+            v-for="v in pastVotings"
+            :key="v.id"
+            :issue="issue"
+          />
+        </QList>
+      </QTabPane>
+    </QTabs>
+  </div>
+</template>
+
+<script>
+import IssueVote from '@/issues/components/IssueVote'
+import IssueResults from '@/issues/components/IssueResults'
+import IssueHistoryItem from '@/issues/components/IssueHistoryItem'
+import ChatConversation from '@/messages/components/ChatConversation'
+import Markdown from '@/utils/components/Markdown'
+import DateAsWords from '@/utils/components/DateAsWords'
+import ProfilePicture from '@/users/components/ProfilePicture'
+
+import {
+  QTabs,
+  QTab,
+  QTabPane,
+  QCollapsible,
+  QList,
+} from 'quasar'
+
+export default {
+  components: {
+    ChatConversation,
+    IssueVote,
+    IssueResults,
+    IssueHistoryItem,
+    QTabs,
+    QTab,
+    QTabPane,
+    QCollapsible,
+    Markdown,
+    DateAsWords,
+    QList,
+    ProfilePicture,
+  },
+  props: {
+    issue: {
+      type: Object,
+      default: null,
+    },
+    conversation: {
+      type: Object,
+      default: null,
+    },
+    away: {
+      type: Boolean,
+      default: false,
+    },
+    inline: {
+      type: Boolean,
+      default: false,
+    },
+    currentUser: {
+      type: Object,
+      default: null,
+    },
+  },
+  computed: {
+    multipleVotings () {
+      return this.issue.votings.length > 1
+    },
+    pastVotings () {
+      return this.issue.votings.filter(v => new Date(v.expiresAt) <= new Date())
```

Comment:
`new Date(...)` can be removed when we extended the `convert` function in the api module

Reply:
...which is not yet, right?